### PR TITLE
Fixed formatting on My Profile page

### PR DESCRIPTION
**Pull request by Boorclark**
Fixes Issue #591 

Changes Made:
 - Got rid of tabs on Service Transcript and My Profile Page
 - Added hyperlink for Service Transcript page
 - Added Table Header
 - Fixed Sizing for headers

### CODE REVIEW

File: app/templates/main/volunteerProfile.html [hunk 1]
```diff
@@ -56,13 +43,17 @@ <h6 align="center">{{volunteer.email}}</h6>
            </tr>
        {% endfor %}
      {% else %}
-       <h4><p class="text-dark">You have no upcoming events. </p></h4>
-       <h4><p class="text-dark">Choose interests below to be notified about your upcoming events.</p><h4>
+     <div class="row">
+       <h4 class="col-sm-10"><p>Upcoming Events: </p></h4>
+       <a class="col-sm-2 nav-link float-end" href="/profile/{{volunteer.username}}/serviceTranscript">View Service Transcript</a>
+     </div>
```
Comment (BrianRamsay) [inline app/templates/main/volunteerProfile.html]: This should be above the upcoming events table

Comment (BrianRamsay) [reply app/templates/main/volunteerProfile.html]: And the text of the link should stay on one line

File: app/templates/main/volunteerProfile.html [hunk 2]
```diff
@@ -56,13 +43,17 @@ <h6 align="center">{{volunteer.email}}</h6>
...
+       <h5><p>You have no upcoming events.</p></h5>
      {% endif %}
    </table>
  </div>
  <!-- Start Program Eligibility Table -->
  <div class="row table-responsive">
+  <caption><h5>Choose interests below to be notified about your upcoming events:</h5></caption>
```
Comment (BrianRamsay) [inline app/templates/main/volunteerProfile.html]: what does the caption tag do? This isn't a caption.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/templates/main/volunteerProfile.html (hunk 1 above)
Comment (Boorclark) [reply app/templates/main/volunteerProfile.html]: resolved

File: app/templates/main/volunteerProfile.html (hunk 2 above)
Comment (Boorclark) [reply app/templates/main/volunteerProfile.html]: resolved

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**